storm_tests/storm_tests.rake

```diff
@@ -17,8 +20,9 @@ COMMON_CFLAGS = %w[
   -Wmissing-declarations
   -Wmissing-noreturn
   -pipe
-  -O3
+  #{OPTIMIZATION_FLAG}
   -funsigned-char
+  -g
```

Comment:
This `-g` was incredibly important to be able to step through the unit test code in VS Code. This link actually helped me realize it: https://github.com/Microsoft/vscode-cpptools/issues/416#issuecomment-274580007